XPCBackupService/Services/BackupUploadService.swift

```diff
@@ -108,16 +135,28 @@ struct BackupUploadService {
             )
 
             self.logger.info("✅ Folder created successfully: \(createdFolder.id)")
+
+            // Save created folder into synced database
+            try self.addSyncedNodeToDB(
```

Comment:
Can we move the Realm logic to another service? `BackupRealm` works, then declare the available methods there